data/guidelines/en/numbers.md

```diff
@@ -1,7 +1,11 @@
 ## Numbers are transcribed as such
 
-Numbers are transcribed as they appear in the document, whether in Roman or Arabic characters. 
-In the former case, ramist standardization (u, v, i, j) must be applied. In the case of numbers written with superscript letters,
-they will be represented by superscript numbers too. We recommend systematically framing them with dots, as is frequently 
-done in manuscripts. The groups of numbers should be clearly separated. 
-The punctuation surrounding the figures should be retained where it exists. 
\ No newline at end of file
+Numbers are transcribed as they appear in the document, whether as Roman or Arabic numerals. In the former case, ramist standardization (u, v, i, j) must be applied as specified in the [dedicated section](./ramists.md) of the guidelines. 
+
+For numbers written with superscript letters, they will be represented by superscript<!--todo: add link to section on superscripted texts--> numbers as well. 
+
+The groups of numbers should be clearly separated. The punctuation surrounding the figures should be retained where it exists. In the case of medieval documents, we recommend systematically framing them with dots, as is frequently done in manuscripts. 
```

Comment:
There is a mistake here. WE SHOULD NOT add dots if there is not dots in the original manuscript.

Reply:
I'll let @matgille see this point before we correct it